docs/pages/reference/cli/tctl.mdx

```diff
@@ -9,6 +9,22 @@ in a cluster, including nodes, users, tokens, certificates, and devices.
 `tctl` can also be used to modify the dynamic configuration of the cluster, such as
 creating new user roles or connecting to trusted clusters.
 
+## Resource kinds
+
+`tctl` commands execute some operation against a Teleport [dynamic
+resource](../../admin-guides/infrastructure-as-code/infrastructure-as-code.mdx).
+For example, the following command retrieves Teleport roles in a cluster:
+
+```code
+$ tctl get role
+```
+
+The following list includes all available Teleport resource kinds:
+
+<Details title="Available resource kinds">
+(!docs/pages/includes/resource_kinds.mdx!)
```

Comment:
Unfortunately, not every `KindXYZ` works with `tctl get`.

The list of accepted values for `tctl get` is defined in a giant switch statement here: https://github.com/gravitational/teleport/blob/8874d074d8a9e9171739701e8b4bfa91ce8f8246/tool/tctl/common/resource_command.go#L2172

Reply:
Thanks! Thought that was too easy 😅